packages/trader/src/Modules/MT5/Containers/mt5-password-modal.jsx

```diff
@@ -84,9 +82,18 @@ const MT5PasswordForm = ({ ...props }) => (
                                         label={localize('Create a password')}
                                         error={touched.password && errors.password}
                                         hint={
-                                            !has_warning &&
-                                            localize(
-                                                'Minimum of eight lower and uppercase English letters with numbers'
+                                            !has_warning && (
+                                                <Localize
+                                                    i18n_default_text='Please confirm your Deriv/Binary.com password to create a DMT5/an MT5 account. <0 /> If you’ve forgotten your password, click <1>Reset password</1>.'
+                                                    components={[
+                                                        <br key={0} />,
+                                                        <NavLink
+                                                            key={1}
+                                                            to={routes.deriv_password}
```

Comment:
is this the right place to send reset password?

Reply:
According to the deisgn, looks like we do have to have a link with a link to the reset-password page.